packages/typography/styles/fonts.scss

```diff
@@ -8,6 +8,19 @@
 		url( 'https://s1.wp.com/i/fonts/recoleta/400.woff' ) format( 'woff' );
 }
 
+@font-face {
+	font-display: swap;
+	font-family: 'RecoletaMediumBold';
+	font-weight: 500;
+	src: url( 'https://s1.wp.com/i/fonts/recoleta/500.woff2' ) format( 'woff2' ),
+		url( 'https://s1.wp.com/i/fonts/recoleta/500.woff' ) format( 'woff' );
+}
+
+.wp-brand-font-medium-bold {
+	font-family: $serif;
+	font-weight: 500;
+}
+
```

Comment:
@ollierozdarz is this `font-weight` change really intended?
I know we recently did some efforts in the opposite direction. Eg: https://github.com/Automattic/wp-calypso/pull/42125
cc: @sixhours

Reply:
Thanks @razvanpapadopol. We chatted about this today in our create-design sync meeting. The intention is to hopefully move towards Recoleta 500 for onboarding related screens, but like you said, it might be wise to make that update all at once down the road. Happy for this to use Recoleta 400 for now!